cg/store/api/find_basic_data.py

```diff
@@ -109,22 +109,35 @@ def get_applications(self) -> List[Application]:
             .all()
         )
 
-    def get_application_versions(self) -> List[ApplicationVersion]:
-        """Return all application versions"""
-        return self._get_query(table=ApplicationVersion).all()
+    def get_application_version_by_application_id(self, application_id: int) -> ApplicationVersion:
```

Comment:
We wanted to be explicit about database entry ids

```suggestion
    def get_application_version_by_application_entry_id(self, application_entry_id: int) -> ApplicationVersion:
```